### ISSUE TITLE
create a financial summary, calculating the Billable deliveries by multiplying cost of 7000 per delivery at the end of every round

### ISSUE THREAD

**Issue by janetbutts** [eHealthAfrica/direct-delivery-dashboard#46]

**Comment by jofomah** [comment]
@janetbutts i think this is not part of landing page but part of financial report.

**Comment by adothompson** [comment]
I think this should be configurable at some level.  Either state, lga, potentially, down to facility level.  And billing may vary we can start with a configurable price per delivery but we may also want to allow a price per km.

**Comment by janetbutts** [comment]
@adothompson , agreed. to make the application scaleable, we should make this configurable. This can go under the configuration page. The financial reporting is not happening in the first release of the software, so we discuss the best process for the next release.
